app.py

```diff
@@ -168,6 +236,22 @@ def main():
         except Exception as e:
             _LOGGER.exception(f"An error occurred pushing the metrics: {str(e)}")
 
+    with _METRIC_RUNTIME.time():
```

Comment:
The reason I did it, so I thought `package_analyzer` refresh syncing jobs should have one metrics like how long it took to sync packages.

Reply:
This will override the previous runtime, place your `graph_refresh_package_analyzer` under the existing timing. Also, please rename `graph_refresh` to something like `graph_refresh_solver`.